examples/3d/chess.rs

```diff
@@ -0,0 +1,482 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .insert_resource(Msaa { samples: 4 })
+        .insert_resource(WindowDescriptor {
+            title: "Chess!".to_string(),
+            width: 1280.0,
+            height: 720.0,
+            ..Default::default()
+        })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_startup_system(create_board)
+        .add_startup_system(create_white_pieces)
+        .add_startup_system(create_black_pieces)
```

Comment:
There's a lot of code duplication between these two systems. Ideally they could be generalized with differing paramaters passed in, that might overcomplicate the example though.

Reply:
IMO showing realistic patterns to reduce code duplication is very important in the more complex game examples. Set a good standard for new users and all that.

Reply:
Each piece when loaded is positioned at the middle of the board. Each team laid out has specific properties associated with it. Since its now just a rendering example, it may seem straight forward to optimize the code once more. The reason why I left it like this, is later when there is more functionality added to `bevy`, I'll be able to add game logic and name the pieces appropriately.

* King
* King's Bishop
* King's Knight
* King's Rook
* Queen
* Queen's Bishop
* Queen's Knight
* Queen's Rook
* Pawn(s)